action/protocol/managers.go

```diff
@@ -96,7 +96,6 @@ type (
 		ProtocolDirty(string) bool
 		Load(string, interface{}) error
 		Unload(string) (interface{}, error)
-		Push() error
```

Comment:
Push() is designed to have multiple workingsets put their items into the Dock, whoever called Push(), his item will be the final value to be committed to Dock

but now with each workingset having its own Dock, Push() is not needed